gpjax/likelihoods.py

```diff
@@ -199,6 +199,42 @@ def predict(self, dist: tfd.Distribution) -> tfd.Distribution:
         return self.link_function(mean / jnp.sqrt(1.0 + variance))
 
 
+@dataclass
+class Beta(AbstractLikelihood):
```

Comment:
You'll need to specify the parameters of the Beta distribution here as we'll seek to learn them during inference. The Beta distribution has two parameters: $\alpha$ and $\beta$, but I think it would be better to parameterise the likelihood in terms of it's mean $\mu = \Phi(f)$ where $\Phi$ is a Gaussian CDF, and dispersion $\nu$. This then makes the shape and scale parameters $\alpha=\nu\mu$ and $\beta=\nu(1-\mu)$. For more details on this, see Section 3.3 of [BOUNDED GAUSSIAN PROCESS REGRESSION](http://www2.imm.dtu.dk/pubdb/edoc/imm6637.pdf)

Reply:
The paper says *v* is an (inverse) dispersion parameter. I'm assuming dispersion refers to standard deviation, but what does it mean to take the inverse of it? Just the reciprocal?